### PR TITLE
New dplyr Cheat Sheet

### PR DESCRIPTION
**Pull request by devarajphukan**
## Description of new Instant Answer, or changes
Dplyr is a library in R to perform various data manipulation tasks with data frame like objects

---

Instant Answer Page: https://duck.co/ia/view/dplyr_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/dplyr.json [hunk 1]
```diff
@@ -0,0 +1,200 @@
+{
+   "id" : "dplyr_cheat_sheet",
+   "name" : "Dplyr Cheat Sheet",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/dplyr.json]: "Dplyr" would be enough as name

File: share/goodie/cheat_sheets/json/dplyr.json [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+   "description" : "Dplyr R basic usage",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/dplyr.json]: Lets make description a little more clear about the contents or give a short description about Dplyr

File: share/goodie/cheat_sheets/json/dplyr.json [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+   ],
+   "template_type": "terminal",
+   "section_order" : [
+      "Basics",
+      "Reshaping Data",
+      "Subset Observations (Row)",
+      "Subset Variables (Columns)",
+      "Summarize Data",
+      "Make New Variables",
+      "Combine Data sets",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/dplyr.json]: Chnage to "Combine Data Sets"

File: share/goodie/cheat_sheets/json/dplyr.json [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+      "Group Data"
+   ],
+   "sections" : {
+      "Basics" : [
+         {
+            "key" : "dplyr::tbl_df(iris)",
+            "val" : "Converts data to tbl class. tbl's are easier to examine than data frames."
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/dplyr.json]: remove periods at end of val fields

File: share/goodie/cheat_sheets/json/dplyr.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
    "id" : "dplyr_cheat_sheet",
-   "name" : "Dplyr Cheat Sheet",
-   "description" : "Dplyr R basic usage",
+   "name" : "Dplyr",
+   "description" : "Dplyr is a library in R to perform various data manipulation tasks with data frame like objects.",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/dplyr.json]: Lets remove period at end of description also

**Comment by jophab** [comment]
There is a problem with ia page.
it should be  https://duck.co/ia/view/dplyr_cheat_sheet

**Comment by devarajphukan** [comment]
@jophab corrected the ia page. Please

**Comment by pjhampton** [comment]
As seen here: https://beta.duckduckgo.com/?q=dplyr+cheat+sheet&ia=cheatsheet
Cool. Thanks, @devarajphukan! 😄 

If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Comment by devarajphukan** [comment]
@pjhampton Thankyou.

**Pull request merged**